.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  building_release:
+    strategy:
+      fail-fast: true
+      matrix:
+        otp_release: ["24", "23", "22"]
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{ matrix.otp_release }}
```

Comment:
Remainder that setup-beam does version resolution, eg `24` would today be resolved as `24.2`. Is this what we want, to always pick latest? I think it is, but perhaps we should leave a comment in case it is not immediately obvious?

Reply:
Added. :)

Reply:
I think we should pin the specific version after all, that's what Bob did for Elixir builds. cc @ericmj

Reply:
Can we use the same version as described in ci.yml:

- 24.0
- 23.3
- 22.3

Reply:
Bob currently has 22.3, 23.3, 24.1, maybe let's use those?

btw, setup-beam allows us to do:

```yaml
otp-version: ...
version-type: strict
```

to pin, so let's do that too!

Reply:
Already push a fix. The CI shows that it uses the strict version https://github.com/wingyplus/elixir/actions/runs/1633938089.